### ISSUE TITLE
Error on training with GPU

### ISSUE THREAD

**Issue by rkakamilan** [rkakamilan/CDN_Molecule#1]
Error with tensorflow-gpu 1.3.0

**Comment by rkakamilan** [comment]
Got the same error with CPU. 

The message was: 
- InvalidArgumentError (see above for traceback): Nan in summary histogram for: fully_connected_2/biases_0/grad/hist

**Comment by rkakamilan** [comment]
Learning rate might be a factor according to the link: https://stackoverflow.com/questions/35976511/issue-with-tensorflow-summary

Will confirm it.

**Comment by rkakamilan** [comment]
NaN error was fixed by adding a small value in tf.log.
